### PR TITLE
fix two typos

### PR DESCRIPTION
**Pull request by ZYFZYF**
Fix two typos
- doc/project3-MultiRaftKV.md:  There should be a newline here
- kv/raftstore/raftstore.go: regionRange shoudle be startKey -> region (if i understand correctly)

### CODE REVIEW

File: kv/raftstore/raftstore.go [hunk 1]
```diff
@@ -37,7 +37,7 @@ func (r *regionItem) Less(other btree.Item) bool {
 
 type storeMeta struct {
 	sync.RWMutex
-	/// region end key -> region ID
+	/// region start key -> region
```
Comment (Connor1996) [inline kv/raftstore/raftstore.go:40]: it's not true

Comment (ZYFZYF) [reply kv/raftstore/raftstore.go:40]: The `Less` of  `regionItem`  :
```
func (r *regionItem) Less(other btree.Item) bool {
	left := r.region.GetStartKey()
	right := other.(*regionItem).region.GetStartKey()
	return bytes.Compare(left, right) < 0
}

```
Isn't this saying that we use startKey as the index of BTree? I'm curious

Comment (Connor1996) [reply kv/raftstore/raftstore.go:40]: Oh, I gets it wrong

**Review by Connor1996 — APPROVED** [review]
LGTM

**Pull request merged**